tools/tpm2_nvwrite.c

```diff
@@ -145,8 +180,11 @@ static bool nv_write(TSS2_SYS_CONTEXT *sapi_context) {
 
 static bool on_option(char key, char *value) {
     bool result;
-
+    ctx.hmac_auth = true;//DEBUG
     switch (key) {
+    case 'X':
```

Comment:
We should use the auth language for this.

```
# compute the hmac ticket
--auth-xxx=hmac:
# use a precomputed ticket
--auth-xxx=hmac:path/to/file
```

Reply:
Implemented